### PR TITLE
Remove unused macros

### PR DESCRIPTION
**Pull request by raveit65**

### CODE REVIEW

File: src/eom-jobs.c [hunk 1]
```diff
@@ -33,8 +33,6 @@
 
 #include <gdk-pixbuf/gdk-pixbuf.h>
 
-#define EOM_JOB_GET_PRIVATE(object) \
-	(G_TYPE_INSTANCE_GET_PRIVATE ((object), EOM_TYPE_JOB, EomJobPrivate))
```
Comment (rbuj) [inline src/eom-jobs.c:36]: There are two consecutive blank lines.

File: src/eom-list-store.c [hunk 2]
```diff
@@ -29,8 +29,6 @@
 
 #include <string.h>
 
-#define EOM_LIST_STORE_GET_PRIVATE(object) \
-	(G_TYPE_INSTANCE_GET_PRIVATE ((object), EOM_TYPE_LIST_STORE, EomListStorePrivate))
```
Comment (rbuj) [inline src/eom-list-store.c:32]: There are two consecutive blank lines.

File: src/eom-scroll-view.c [hunk 3]
```diff
@@ -19,9 +19,6 @@
 
 #include <gdk/gdk.h>
 
-/* Maximum size of delayed repaint rectangles */
-#define PAINT_RECT_WIDTH 128
-#define PAINT_RECT_HEIGHT 128
```
Comment (rbuj) [inline src/eom-scroll-view.c:22]: There are two consecutive blank lines.

File: src/main.c [hunk 4]
```diff
@@ -46,7 +46,6 @@
 #include <exempi/xmp.h>
 #endif
 
-#define EOM_CSS_FILE_PATH EOM_DATA_DIR G_DIR_SEPARATOR_S "eom.css"
```
Comment (rbuj) [inline src/main.c:49]: There are two consecutive blank lines.

**Review by rbuj — COMMENTED** [review]
Everything works fine as before, but there are new consecutive blank lines.

File: src/eom-image.c [hunk 5]
```diff
@@ -61,9 +61,6 @@
 
 #if defined(HAVE_LCMS) && defined(GDK_WINDOWING_X11)
 #include <lcms2.h>
-#ifndef EXIF_TAG_GAMMA
-#define EXIF_TAG_GAMMA 0xa500
-#endif
```
Comment (rbuj) [inline src/eom-image.c:66]: EXIF_TAG_GAMMA was added on libexif-0.6.22, eom requires libexif-0.6.14 ([REF](https://github.com/libexif/libexif/blob/303643aebb75b2a9f3db83b774b1d448fdfe694b/NEWS#L12))
```
$ git grep LIBEXIF_REQUIRED=
configure.ac:LIBEXIF_REQUIRED=0.6.14
```
This macro is used in another file, but it doesn't check whether it's defined:
```
$ git grep -n EXIF_TAG_GAMMA
src/eom-metadata-reader-jpg.c:608:                      entry = exif_data_get_entry (exif, EXIF_TAG_GAMMA);
```

Comment (raveit65) [reply src/eom-image.c:66]: Libexif required version is updated.

**Comment by raveit65** [comment]
All done.

**Review by rbuj — APPROVED** [review]
LGTM

**Pull request merged**